### PR TITLE
Create editor for categories

### PR DESCRIPTION
**Pull request by Adelhaide22**

### CODE REVIEW

File: Core/Helpers/StateManager.cs [hunk 1]
```diff
@@ -84,6 +97,18 @@ public static void UpdateTransaction(Transaction transaction)
...
 
+        public static void UpdateCategory(Category category)
+        {
+            var categories = State.Instance.Categories.ToHashSet();
+            if (!categories.Contains(category))
+            {
+                throw new ArgumentException($"{nameof(category)} not found");
+            }
+            categories.Remove(category);
+            categories.Add(category);
```
Comment (siriak) [inline Core/Helpers/StateManager.cs:108]: ?

File: WinFormsUI/AddCategoryForm.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+namespace WinFormsUI
+{
+    public partial class AddCategoryForm : Form
+    {
+        public AddCategoryForm()
+        {
+            InitializeComponent();
+        }
+
+        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
```
Comment (siriak) [inline WinFormsUI/AddCategoryForm.cs]: comboBox1

File: WinFormsUI/DisplayManager.cs [hunk 3]
```diff
@@ -21,5 +24,52 @@ public static string GetPrefix(Levels level)
                 _ => "(FULL) ",
             };
         }
+
+        public static IReadOnlyCollection<string> ReadList(string text)
+        {
+            return text.Split("\r\n");
+        }
+
+        public static string DisplayList(IReadOnlyCollection<string> text)
```
Comment (siriak) [inline WinFormsUI/DisplayManager.cs]: Display doesn't display

File: Core/Helpers/StateManager.cs (hunk 1 above)
Comment (Adelhaide22) [reply Core/Helpers/StateManager.cs:108]: Remove - removes old element, add - adds updated

**Review by siriak — APPROVED** [review]

**Pull request merged**